tests/test_json.py

```diff
@@ -9,6 +10,11 @@
 from flask import json
 
 
+def test_json_encode_decimal():
+    decimal_ = decimal.Decimal(100)
```

Comment:
how about 0.03 instead of 100? :)

Reply:
fyi django cast decimal to str instead of float https://github.com/django/django/blob/225d96533a8e05debd402a2bfe566487cc27d95f/django/core/serializers/json.py#L103

Reply:
Hey, are you sure you want it to be in str ? Cuz I initially casted it to string but ThiefMaster said its better to have it as a float.

Reply:
Thief said "not int" not "float". It should definitely be str, that's the only way to not lose precision (at the expense of losing the number type).